packages/react/src/components/DataGrid/index.tsx

```diff
@@ -282,6 +282,13 @@ export const DataGrid = <T extends Row>({
                         } else {
                           highlight = !!columns[i].highlight;
                         }
+                        const style: CSSProperties = {};
```

Comment:
I tried to see if it is possible to simply hide the headers from CSS and keep the width at the same time, but I did not succeed 😅

So what do you think about simply setting the width of the cells whether we displayHeader or not ? That way we will have one way to achieve it?

Reply:
Agree. I just find our solution more elegant and would like to keep it when it was possible but indeed it duplicates code that is weird.

Reply:
I tried to only set width on row cell but when there is a header it does not work, we have also to set width header cells... So finally I'll do nothing about that.